website/src/index.html

```diff
@@ -376,6 +312,10 @@ <h2>Track 3</h2>
               We strive to closely simulate real-world dynamics.
               But, no simulator is perfect.
               Thus, we invite you to point out potential improvements and loopholes.
+              Please see the <a
```

Comment:
for <suggested> aspects

Reply:
I didn't understand this one. Should I remove "to discuss"?

Reply:
oops, I meant "for suggestions on what aspects to discuss."